src/Dhadgar.Console/CLAUDE.md

```
@@ -5,17 +5,22 @@ Real-time server console streaming.
 ## Tech Stack
 - ASP.NET Core
 - SignalR for real-time communication
+- PostgreSQL with EF Core
+- Redis for session management
 
 ## Port
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)